data/eminance-ways.json

```diff
@@ -3,6 +3,8 @@
   "establishedAt": 1364307090,
   "location": "Maitidevi, Kathmandu",
   "website": "https://eminenceways.com/",
+  "email": "info@eminenceways.com",
+  "phone": "+977-014569328",
```

Comment:
I guess the phone number format should be `+977-1` instead of `+977-01`

Reply:
Alright, fixing it in all the files then 🙇